diff --git a/node_modules/y-webrtc/dist/y-webrtc.cjs b/node_modules/y-webrtc/dist/y-webrtc.cjs
index 71d0ec4..dcb089c 100644
--- a/node_modules/y-webrtc/dist/y-webrtc.cjs
+++ b/node_modules/y-webrtc/dist/y-webrtc.cjs
@@ -647,7 +647,9 @@ class SignalingConn extends ws.WebsocketClient {
                 if (data.signal.type === 'answer') {
                   log('offer answered by: ', data.from);
                   const existingConn = webrtcConns.get(data.from);
-                  existingConn.glareToken = undefined;
+                  if (existingConn) {
+                    existingConn.glareToken = undefined;
+                  }
                 }
                 if (data.to === peerId) {
                   map.setIfUndefined(webrtcConns, data.from, () => new WebrtcConn(this, false, data.from, room)).peer.signal(data.signal);
diff --git a/node_modules/y-webrtc/src/y-webrtc.js b/node_modules/y-webrtc/src/y-webrtc.js
index 609328a..ad55d3a 100644
--- a/node_modules/y-webrtc/src/y-webrtc.js
+++ b/node_modules/y-webrtc/src/y-webrtc.js
@@ -536,7 +536,9 @@ export class SignalingConn extends ws.WebsocketClient {
                 if (data.signal.type === 'answer') {
                   log('offer answered by: ', data.from)
                   const existingConn = webrtcConns.get(data.from)
-                  existingConn.glareToken = undefined
+                  if (existingConn) {
+                    existingConn.glareToken = undefined
+                  }
                 }
                 if (data.to === peerId) {
                   map.setIfUndefined(webrtcConns, data.from, () => new WebrtcConn(this, false, data.from, room)).peer.signal(data.signal)
